.circleci/config.yml

```diff
@@ -54,6 +16,10 @@ jobs:
           command: |
             git submodule init
             git submodule update
+      - restore_cache:
```

Comment:
So is the idea that this will prevent multiple model/wrapper builds when users approve running multiple sets of tests?

Reply:
That's correct.  It doesn't save a huge amount of time (maybe about 2 minutes or so) but it seems like we might as well take advantage of it if we can.